docset/winserver2022-ps/grouppolicy/Get-GPO.md

```diff
@@ -16,32 +16,42 @@ Gets one GPO or all the GPOs in a domain.
 ## SYNTAX
 
 ### ByGUID (Default)
+
 ```
 Get-GPO [-Guid] <Guid> [[-Domain] <String>] [[-Server] <String>] [-All] [<CommonParameters>]
 ```
 
 ### ByName
+
 ```
 Get-GPO [-Name] <String> [[-Domain] <String>] [[-Server] <String>] [-All] [<CommonParameters>]
 ```
 
 ### GetAll
+
 ```
 Get-GPO [[-Domain] <String>] [[-Server] <String>] [-All] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
+
 The **Get-GPO** cmdlet gets one Group Policy Object (GPO) or all the GPOs in a domain.
```

Comment:
We always want to format cmdlets as code with backticks.
```suggestion
The `Get-GPO` cmdlet gets one Group Policy Object (GPO) or all the GPOs in a domain.
```

Reply:
Gotcha!